client/components/Home.jsx

```diff
@@ -6,20 +6,23 @@ import {
   addSelectedImage,
   removeSelectedImage,
   getSingleImage,
+  flipSelect,
 } from '../redux/images';
+import Tag from './Tag.jsx';
+import SingleImage from './SingleImage.jsx';
 
-//select button to changes between single image veiw and select images
-
-const Home = ({ images }) => {
+const Home = props => {
+  const { images } = props;
   return (
     <div
       style={{
         display: 'flex',
       }}
     >
       <h1
-        onClick={() => {
-          this.props.select = !this.props.select;
+        onClick={event => {
+          event.preventDefault();
+          props.swapSelect();
```

Comment:
Doesn't look like swapSelect was brought in from props above, probably will throw error.

Reply:
It doesn't need to be pulled out if you use props.whatever. But I'll pull them out up top with images for uniformity.

Reply:
Pulled everything out of props.